packages/http-client-js/src/components/operation-handlers/paginated-operation-handler.tsx

```diff
@@ -0,0 +1,191 @@
+import * as ay from "@alloy-js/core";
```

Comment:
I think we should move this inside paging folder

Reply:
Yeah, this is a better place.